config/tools/websocket-source/websocket-source.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is a very simple Knative Eventing Source for consuming WebSocket (Secure) messages.
+apiVersion: sources.knative.dev/v1alpha2
+kind: ContainerSource
+metadata:
+  name: websocket-source
+spec:
+  image: ko://knative.dev/eventing-contrib/cmd/websocketsource
+  sink:
+    apiVersion: serving.knative.dev/v1
+    kind: Service
+    name: event-display
```

Comment:
Do you want to call it `event-display` or this was overlooked?

Reply:
missing ref?

```
sink:
  ref:
    ...
```